src/components/molecules/RadioPopoverContent/RadioPopoverContent.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import { getRadioStationUrl } from "utils/radio";
+
+import { RadioModal } from "components/organisms/RadioModal/RadioModal";
+
+import "./RadioPopoverContent.scss";
+
+export interface RadioPopoverContentProps {
+  radioTitle: string;
+  showNormalRadio: boolean;
+  showSoundCloudRadio: boolean;
+  radioStation: string | false;
+  isRadioPlaying: boolean;
+  handleRadioEnable: () => void;
+  volume: number;
+  setVolume: React.Dispatch<React.SetStateAction<number>>;
+}
+
+export const RadioPopoverContent: React.FC<RadioPopoverContentProps> = ({
+  radioTitle,
+  showNormalRadio,
+  showSoundCloudRadio,
+  radioStation,
+  isRadioPlaying,
+  handleRadioEnable,
+  volume,
+  setVolume,
+}) => (
+  <>
+    <div className="RadioPopoverContent--title">{radioTitle}</div>
```

Comment:
From BEM style `--title` is here a modifier when it should probably be block and/or element

Reply:
fixed 433dfe8